msal/tests/Test.MSAL.NET.Unit.net45/Resources/OpenidConfiguration-B2CLogin.json

```diff
@@ -0,0 +1,17 @@
+﻿{
+  "authorization_endpoint": "https://mytenant.com.b2clogin.com/6babcaad-604b-40ac-a9d7-9fd97c0b779f/oauth2/v2.0/authorize?p=my-policy",
+  "token_endpoint": "https://mytenant.com.b2clogin.com/6babcaad-604b-40ac-a9d7-9fd97c0b779f/oauth2/v2.0/token?p=my-policy",
```

Comment:
>my-policy [](start = 112, length = 9)

ah, this is likely what needs to be updated. the policy should be part of the url right after the tenant afaik

Reply:
i fixed this, but one of the conf client tests is checking for the policy as a qp, so I made another json file for that case. I didn't want to change the unit test at this point. also, i noticed these files have to be in the adal tests too in order for the utests to pass...which seems odd. I'll sync w/bogdan on that.

---
In reply to: [234397668](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1405#discussion_r234397668) [](ancestors = 234397668)